webcompat/webhooks/model.py

```diff
@@ -185,12 +189,16 @@ def close_public_issue(self, reason='rejected'):
         Right now the accepted reasons are:
         'incomplete'
         'invalid'
+        'autoclosed'
         'rejected' (default)
         """
         if reason == 'incomplete':
             payload_request = self.prepare_accepted_issue('incomplete')
         elif reason == 'invalid':
             payload_request = self.prepare_accepted_issue('invalid')
+        elif reason == 'autoclosed':
+            payload_request = prepare_rejected_issue(reason)
```

Comment:
Probably?

```suggestion
            payload_request = self.prepare_rejected_issue(reason)
```

Reply:
I've reused `prepare_rejected_issue` here, which is an existing function in `webhooks_helpers.py` file, so this is correct